tests/store/api/find/test_find_business_data.py

```diff
@@ -731,3 +733,45 @@ def test_get_case_by_name_and_customer_case_found(store_with_multiple_cases_and_
     assert filtered_case is not None
     assert filtered_case.customer_id == customer.id
     assert filtered_case.name == case.name
+
+
+def test_get_cases_not_analysed_by_sample_internal_id_empty_query(
+    store_with_multiple_cases_and_samples: Store,
+):
+    """Test that an empty query is returned if no cases match the sample internal id."""
+    # GIVEN a store with no cases matching the sample internal id
+    sample_internal_id = "nonexistent_sample_internal_id"
```

Comment:
there is a fixture in `test/conftest.py` called `fixture_invalid_sample_id` that maybe you can use here